src/backend/functions/change-request-new.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import middy from '@middy/core';
+import jsonBodyParser from '@middy/http-json-body-parser';
+import httpErrorHandler from '@middy/http-error-handler';
+import validator from '@middy/validator';
+import { Handler as NetlifyHandler } from '@netlify/functions';
+import { Handler } from 'aws-lambda';
+import { PrismaClient, Prisma, CR_Type, Scope_CR_Why_Type } from '@prisma/client';
+import { buildSuccessResponse } from 'utils';
+// TODO: clean up imports
+
+const prisma = new PrismaClient();
+
+// TODO: add a comment to explain the function?
+const createStandardChangeRequest = async (
+  submitterId: number,
+  wbsElementId: number,
+  type: CR_Type,
+  body: {
+    what: string;
+    scopeImpact: string;
+    timelineImpact: number;
+    budgetImpact: number;
+    why: { explain: string; type: Scope_CR_Why_Type }[];
+  }
+) => {
+  const createdChangeRequest = await prisma.change_Request.create({
+    data: {
+      submitter: { connect: { userId: submitterId } },
+      wbsElement: { connect: { wbsElementId } },
+      type,
+      scopeChangeRequest: {
+        create: {
+          what: body.what,
+          scopeImpact: body.scopeImpact,
+          timelineImpact: body.timelineImpact,
+          budgetImpact: body.budgetImpact,
+          why: { createMany: { data: body.why } }
+        }
+      }
+    }
+  });
+  // TODO: figure out what is best to return
+  return buildSuccessResponse(createdChangeRequest);
+};
+
+// TODO: add a comment to explain the function?
+const baseHandler: Handler = async ({ body }, _context) => {
+  const { submitterId, wbsElementId, type } = body;
+  let created;
+  switch (type) {
+    case CR_Type.ACTIVATION:
+      // TODO: create the function
+      break;
+    case CR_Type.STAGE_GATE:
+      // TODO: create the function
+      break;
+    case CR_Type.OTHER:
+      created = await createStandardChangeRequest(submitterId, wbsElementId, type, body);
+      break;
+    case CR_Type.ISSUE:
+      created = await createStandardChangeRequest(submitterId, wbsElementId, type, body);
+      break;
+    case CR_Type.DEFINITION_CHANGE:
+      created = await createStandardChangeRequest(submitterId, wbsElementId, type, body);
+      break;
+    default:
+      // TODO: create the else case fail
+      break;
+  }
+  // TODO: change this return statement
+  return buildSuccessResponse(created || { result: 'success', message: 'cr created correctly' });
+};
+
+// validates the event parameter, so body must be explicitly stated
+// TODO: actually complete the schema
+// TODO: consider moving schemas (or parts of schemas) to utils package? consider using schema-to-ts?
+const inputSchema = {
```

Comment:
Note: the validation schema I have here is definitely a work-in-progress. I was examining how to build the complex typing of a change request into JSON schema and simply didn't have enough time to figure it out since I wanted to move forward with this PR. Here's some of what I was looking at:

https://stackoverflow.com/questions/9029524/json-schema-specify-field-is-required-based-on-value-of-another-field

Reply:
Currently, bad input values seems to displease Prisma in ways I don't fully understand, so I think it's likely wisest for us to be validating all the inputs. I also think this syntax is kinda nice for defining input constraints.

Reply:
Your example of `GET /change-requests/word` refers to a different endpoint though, no?

Reply:
Yes, it's a different endpoint, but reinforcing that all user sent requests need to be validated